.evergreen/config.yml

```diff
@@ -380,6 +380,7 @@ functions:
       params:
         working_dir: "src"
         shell: bash
+        include_expansions_in_env: true
```

Comment:
IIUC this should be a list of vars to avoid pulling in vars we don't need.

Reply:
According to the mitigation document, this is an appropriate solution for now.

Reply:
Updating to use add_expansions_to_env, which we have used in several places.  I'll open up a follow-up ticket to make them all explicit.

Reply:
Wasn't there an issue where add_expansions_to_env caused AWS env vars to be injected thus causing issues in the MONGODB-AWS tests? If not I'm fine with your approach.